libgearman-server/plugins/queue/redis/queue.cc

```diff
@@ -118,13 +118,13 @@ gearmand_error_t Hiredis::initialize()
   int service_port= atoi(service.c_str());
   if ((_redis= redisConnect("127.0.0.1", service_port)) == NULL)
   {
-    return gearmand_gerror("Could not connect to redis server", GEARMAND_QUEUE_ERROR);
+    return gearmand_log_gerror(GEARMAN_DEFAULT_LOG_PARAM, GEARMAND_QUEUE_ERROR, "Could not connect to redis server: %s", _redis->errstr);
```

Comment:
Can you please make this a bit more vertical? It's very wide, which makes it a bit confusing to read even with a very wide editor.